src/backend/src/routes/slack.routes.ts

```diff
@@ -0,0 +1,12 @@
+import { createEventAdapter } from '@slack/events-api';
+import SlackController from '../controllers/slack.controllers';
+
+export const slackEvents = createEventAdapter(process.env.SLACK_SIGNING_SECRET || '');
+
+slackEvents.on('message', async (event) => {
```

Comment:
you can remove the async (event) and just have it be SlackController.processsMessageEvent